### PR TITLE
Data deactivate popup

### PR DESCRIPTION
**Pull request by OjasA1**

### CODE REVIEW

File: includes/class-gdpr-cookie-consent.php [hunk 1]
```diff
@@ -315,6 +324,90 @@ public function get_loader() {
...
+		);
+		wp_localize_script(
+			'gdpr-cookie-consent-admin-deactivatePopup',
+			'gdpr_localize_deactivatePopup_data',
+			array(
+				'ajaxurl'           => admin_url( 'admin-ajax.php' ),
+				'_ajax_nonce'       => wp_create_nonce( 'gdpr-cookie-consent' ),
+			)
+		);
+		add_action( 'admin_footer', [ $this, 'print_deactivate_deactivatePopup_dialog' ] );
```
Comment (Omendra123Rajput) [inline includes/class-gdpr-cookie-consent.php:365]: change callback function name

File: includes/class-gdpr-cookie-consent.php [hunk 2]
```diff
@@ -315,6 +324,90 @@ public function get_loader() {
...
+            <div class="gdpr-deactivatePopup-form-wrapper">
+                <form class="gdpr-deactivatePopup-form">
+                    <div>
+                    <p class="deactivatePopup-form-title">Deactivate WP Cookie Consent :</p>
+					<div class="deactivatePopup-form-description">
+					<p class="deactivatePopup-form-description-content">You are about to deactivate WP Cookie Consent. Would you like to delete its data or keep it in place?</p>
+					</div>
+                    <div class="deactivatePopup-inputs">
+                      <div class="input-choices">
+                        <input type="radio" id="without_data" name="reason" value="without_data">
```
Comment (Omendra123Rajput) [inline includes/class-gdpr-cookie-consent.php:388]: give id proper name, avoid using generic name

File: includes/class-gdpr-cookie-consent.php [hunk 3]
```diff
@@ -315,6 +324,90 @@ public function get_loader() {
...
+                    <label for="without_data">Keep all WP Cookie Consent tables and data</label><br>
```
Comment (Omendra123Rajput) [inline includes/class-gdpr-cookie-consent.php:389]: give id proper name, avoid using generic name

File: includes/class-gdpr-cookie-consent.php [hunk 4]
```diff
@@ -315,6 +324,90 @@ public function get_loader() {
...
+                      </div>
+                      <div class="input-choices">
```
Comment (Omendra123Rajput) [inline includes/class-gdpr-cookie-consent.php:391]: give class proper name, avoid using generic name

File: admin/class-gdpr-cookie-consent-admin.php [hunk 5]
```diff
@@ -191,7 +191,68 @@ public function wpl_cookie_consent_integrations_settings() {
...
+	 * 2 Ajax function callback method.
+	 */
+	public static function gdpr_cookie_consent_deactivatePopup(){
+		// Verify AJAX nonce.
+		check_ajax_referer( 'gdpr-cookie-consent', '_ajax_nonce' );
+		
+		if ( ! current_user_can( 'activate_plugins' ) ) {
+			wp_send_json_error( 'Permission denied' );
+		}
+		if (!empty($_POST) && isset($_POST['reason'])) {
```
Comment (Omendra123Rajput) [inline admin/class-gdpr-cookie-consent-admin.php:208]: Run phpcbf for the space alginments in the code

File: admin/css/wpl-cookie-consent-deactivatePopup.css [hunk 6]
```diff
@@ -0,0 +1,112 @@
+/**
+ * All of the CSS for your deactivate popup functionality should be
+ * included in this file.
+ */
+
+.gdpr-deactivatePopup-form-wrapper-outer{
```
Comment (Nayan-Gaikwad) [inline admin/css/wpl-cookie-consent-deactivatePopup.css:6]: @OjasA1 Please use proper classnames all over

**Review by Nayan-Gaikwad — CHANGES_REQUESTED** [review]

**Review by Nayan-Gaikwad — APPROVED** [review]

**Pull request merged**